roles/setup_hammerdbserver/defaults/main.yml

```diff
@@ -0,0 +1,14 @@
+---
+hammerdb_version: 4.6
+hammerdb_filename: "HammerDB-{{ hammerdb_version }}-\
+                    {{ 'RHEL' if ansible_os_family == 'RedHat' else 'Linux' }}\
+                    {{ ansible_distribution_major_version if \
+                    ansible_os_family == 'RedHat'}}.tar.gz"
+hammerdb_url: "https://github.com/TPC-Council/HammerDB/releases/download/"
+supported_os:
+  - CentOS8
+  - RHEL8
+  - Rocky8
```

Comment:
Please undo these changes on this file. I think you just have to rebase your branch.

Reply:
I removed the file and rebased it.
Sorry for the confusion.

Reply:
Looks good. Could you please update the main README file to reflect that almalinux8 is now supported? Thank you.

Reply:
I added the information into the main README file as suggested.